src/test/java/uk/gov/hmcts/reform/slc/services/steps/maptoletter/LetterMapperTest.java

```diff
@@ -42,15 +43,51 @@ public void should_return_letter_if_message_contains_valid_json() {
         assertThat(letter).isNotNull();
         assertThat(letter.template).isEqualTo("whatever");
         assertThat(letter.type).isEqualTo("some_type");
+        assertThat(letter.service).isEqualTo("some_service");
     }
 
-
     @Test
     public void should_throw_an_exception_if_message_contains_invalid_json() {
         given(message.getBody())
             .willReturn("{\"a\" : \"b\"}".getBytes());
 
         assertThatThrownBy(() -> letterMapper.from(message))
-            .isInstanceOf(InvalidMessageException.class);
+            .isInstanceOf(InvalidMessageException.class)
+            .hasMessageStartingWith("Unable to deserialize message");
+    }
+
+    @Test
+    public void should_throw_an_exception_if_message_contains_null_fields() {
```

Comment:
`{}` contains null fields?

Reply:
All fields unmarshall as null. Don't want to prove the case with `4!` attempts for 3 scenarios just to make point code branch works

Reply:
I mean that method description doesn't match value used in test